build/configs/artik053/artik053_download.sh

```diff
@@ -47,6 +47,13 @@ main()
 		case ${arg} in
 		ALL)
 			echo "ALL :"
+
+			# check existence of os binary
+			if [ ! -f "${OUTPUT_BINARY_PATH}/tinyara_head.bin" ]; then
```

Comment:
No. ALL option should be used for all binaries including firmware. All firmware has to be checked then.